src/table/components/TableWrapper.jsx

```diff
@@ -247,7 +159,16 @@ export default function TableWrapper(props) {
           />
         </Table>
       </TableContainer>
-      {paginationBar}
+      {!constraints.active && (
```

Comment:
As far as I understand we always want to hide the footer in edit mode.

Reply:
@T-Wizard Could you please have a look?

Reply:
sounds good